packages/@okta/vuepress-site/docs/reference/telephony-hook/index.md

```diff
@@ -16,15 +16,17 @@ This information is specific to the Telephony Inline Hook, one type of Inline Ho
 
 ## See also
 
-For a general introduction to Okta Inline Hooks, see [Inline Hooks](/docs/concepts/inline-hooks/).
+- For a general introduction to Okta Inline Hooks, see [Inline Hooks](/docs/concepts/inline-hooks/).
 
-For information on the API for registering external service endpoints with Okta, see [Inline Hooks Management API](/docs/reference/api/inline-hooks/).
+- For information on the API for registering external service endpoints with Okta, see [Inline Hooks Management API](/docs/reference/api/inline-hooks/).
```

Comment:
I would also include a link to the h.o.c. and how to implement a hook in admin console. I've never used the API. :o)

https://help.okta.com/oie/en-us/Content/Topics/automation-hooks/add-inline-hooks.htm

see Inline Hooks Management API or Add an Inline Hook.